Generator/JobGenerator.php

```diff
@@ -14,20 +14,25 @@
  * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
-class JobGenerator
+class JobGenerator implements GeneratorInterface
 {
+    const TYPE = 'jobs';
+
     const JOB_FILENAME = 'jobs.yml';
 
     const INTERNAL_JOBS_FILE = 'Resources/config/internal_jobs.yml';
```

Comment:
Again, Having formats sticked to the generators can become a problem in the long run.